Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -110,21 +118,35 @@ public SiteExtensionManager(IContinuousJobsManager continuousJobManager, ITrigge
         {
             ITracer tracer = _traceFactory.GetTracer();
 
-            SourceRepository remoteRepo = GetRemoteRepository(feedUrl);
+            IEnumerable<SourceRepository> remoteRepos = GetRemoteRepositories(feedUrl);
             UIPackageMetadata package = null;
 
             if (string.IsNullOrWhiteSpace(version))
             {
                 using (tracer.Step("Version is null, search latest package by id: {0}", id))
                 {
-                    package = await remoteRepo.GetLatestPackageByIdFromSrcRepo(id);
+                    foreach (SourceRepository remoteRepo in remoteRepos)
+                    {
+                        package = await remoteRepo.GetLatestPackageByIdFromSrcRepo(id);
```

Comment:
Not sure this is quite correct, as this will return any package from nuget.org, even if it doesn't have our `AzureSiteExtension` tag.

Reply:
Ah, you're right.  Only [.Search()](https://github.com/projectkudu/kudu/blob/150dab5ff20fe525e7eb64f9629a64664adb7cb3/Kudu.Core/SiteExtensions/FeedExtensions.cs#L39-L59) has the special logic to automagically add the AzureSiteExtension magic tag.  I'll have to figure out the best place to put this magic to avoid duplicating code everywhere.

Reply:
Side Question: is "AzureSiteExtension" case sensitive?  And do we need to worry about collisions?
 e.g. "DefinitelyNotAnAzureSiteExtension" vs. "AzureSiteExtension"

Reply:
Not sure about the casing, but I highly doubt that it is sensitive (you'll have to try it on NuGet.org with the one package we have there).

No, I'm not worried about those collisions. The tag is unique enough that it won't happen accidentally.